agent/pyproject.toml

```diff
@@ -49,6 +64,7 @@ build-backend = "poetry.core.masonry.api"
 
 [project.scripts]
 crystaldba = "crystaldba.cli.main:main"
```

Comment:
Let's go ahead and replace `crystaldba` with the `elia` implementation.

Let's keep the the old one around as `crystaldba-cli-0.8`